CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.18.2 - 2021-07-30
+### Fixed
+- [Fix for Timing Out issue when loading Postgres Flexible Servers](https://github.com/microsoft/vscode-cosmosdb/issues/1913)
```

Comment:
Instead of making this whole sentence a link, I'd recommend doing something like [this](https://github.com/microsoft/vscode-azurestorage/blob/main/CHANGELOG.md#fixed-7) and link just the issue at the end:

```suggestion
- Fix time out issue when loading Postgres Flexible Servers [#1913](https://github.com/microsoft/vscode-cosmosdb/issues/1913)
```

Reply:
https://github.com/microsoft/vscode-azurefunctions/blob/main/CHANGELOG.md#0191---2019-10-15

For reference, this is how Eric usually does it